content/articles/support-vector-regression-in-python/index.md

```diff
@@ -0,0 +1,303 @@
+### Getting Started with Support Vector Regression in Python
+Support Vector Regression, denoted as SVR, is a regression function that generalizes the finite Support Vector Machines model on a continuous-valued output. Although less popular in real-valued function estimation, this algorithm has proven very effective.
+
+### Prerequsites:
+The learner is required to have a good understanding of:
+- Ordinary Least Squares method
+- [Lagrangian multiplier method](https://www.youtube.com/watch?v=5m32gu6qGIU&t=809s)
+- How to work with pandas, numpy, sk-learn, and matplotlib libraries in python.
+
+### Introduction to Support Vector Regression
+To understand how the SVR model works, let's first recap the simple linear regression. In linear regression, we represent the regression line as:
```

Comment:
While recapping, please use a definition or explain it in words before directly jumping into visual representation using graphs.